doc/UserGuide/tables/UPP_GRIB2_Table_byID.csv

```diff
@@ -699,29 +700,30 @@ Dust on pressure surface,DUST_ON_ISOBARIC_SFC,MASSDEN,743
 Biomass burning emissions,BIOMASS_BURNING_EMISSIONS,AEMFLX,745
 Accumulated graupel,ACM_GRAUPEL_ON_SURFACE,FROZR,746
-Number concentration for cloud water drops on hybrid level,GSD_NCCD_ON_HYBRID_LVL,NCONCD,747
+Number concentration for cloud water drops on hybrid level,NCCD_ON_HYBRID_LVL,NCONCD,747
 Radar reflectivity at 1km above ground,GSD_REFL_ON_SPEC_HGT_LVL_ABOVE_GRND_1km,REFD,748
 Relative humidity with respect to precipitable water,GSD_RH_WRT_PRECIP_WATER_ON_ENTIRE_ATMOS,RHPW,749
 Water vapor mixing ratio on model surface,GSD_WV_MIXR_ON_HYBRID_LVL,MIXR,750
 Virtual potential temperature on model surface,GSD_VPTMP_ON_HYBRID_LVL,VPTMP,751
-Number concentration for ice particles on hybrid level,GSD_NCIP_ON_HYBRID_LVL,NCCICE,752
+Number concentration for ice particles on hybrid level,NCIP_ON_HYBRID_LVL,NCCICE,752
 GSD pressure on level of 0 deg (C) isotherm,GSD_PRES_ON_0C_ISOTHERM,PRES,753
-Number concentration of rain drops on model surface,GSD_NCRAIN_ON_HYBRID_LVL,SPNCR,754
-Hourly Wildfire Potential on surface,HWP_ON_SURFACE,FWINX,755
+Number concentration of rain drops on model surface,NCRAIN_ON_HYBRID_LVL,SPNCR,754
+Hourly Wildfire Potential on surface,HWP_ON_SURFACE,WFIREPOT,755
 GSD pressure on highest tropospheric freezing level,GSD_PRES_ON_HGHST_TROP_FRZ_LVL,PRES,756
 GSD radar reflectivity at 4km above ground,GSD_REFL_ON_SPEC_HGT_LVL_ABOVE_GRND_4km,REFD,757
 GSD convective cloud top height,GSD_HGT_ON_CONVECTIVE_CLOUD_TOP_LVL,HGT,758
+Hourly averaged smoke particulate matter 2.5 (PM) at 8m above ground,AVE_PM25_ON_SPEC_HGT_LVL_ABOVE_GRND_8m,MASSDEN,759  
 GSD shelter mixing ratio,GSD_MIXR_ON_SPEC_HGT_LVL_ABOVE_GRND_2m,MIXR,760
 GSD snow temperature,GSD_INSIDE_SNOW_TMP_ON_SURFACE,TMP,761
 GSD surface mixing ratio,GSD_MIXR_ON_SURFACE,MIXR,762
 GSD mixing ration on level of free convection (LFC),GSD_MIXR_ON_LFC,MIXR,763
 Instantaneous clear-sky downwelling longwave at the surface,CSDLF_ON_SURFACE,CSDLF,764
 Instantaneous clear-sky upwelling longwave at the surface,CSULF_ON_SURFACE,CSULF,765
-GSD water-friendly aerosol number concentration on model surface,GSD_NCWFA_ON_HYBRID_LVL,PMTF,766
-GSD ice-friendly aerosol number concentration on model surface,GSD_NCIFA_ON_HYBRID_LVL,PMTC,767
+GSD water-friendly aerosol number concentration on model surface,NCWFA_ON_HYBRID_LVL,PMTF,766
+GSD ice-friendly aerosol number concentration on model surface,NCIFA_ON_HYBRID_LVL,PMTC,767
 Echo top height (Highest height in meters of the 18-dBZ reflectivity on a model level),GSD_ECHOTOP_ON_CLOUD_TOP,RETOP,768
-GSD Vertically integrated liquid in kg/m^2,GSD_VIL_ON_ENTIRE_ATMOS,VIL,769
-GSD Vertically integrated liquid based on reflectivity factor in kg/m^2,GSD_RADARVIL_ON_ENTIRE_ATMOS,RADARVIL,770
+GSD Vertically integrated liquid in kg/m^2 dervied from hydrometer mass,GSD_VIL_ON_ENTIRE_ATMOS,VIL,769
+Hourly averaged dust particulate matter 1.0 (PM) at 8m above ground,AVE_DUST_ON_SPEC_HGT_LVL_ABOVE_GRND_8m,MASSDEN,771
```

Comment:
```suggestion
Hourly averaged dust 10-micron particulate matter (PM10) at 8m above ground,AVE_PM10_ON_SPEC_HGT_LVL_ABOVE_GRND_8m,MASSDEN,771
```
@ChristopherHill-NOAA Can you confirm the field description or suggest a change?

Reply:
The new field description and parameter name for variable 771 will provide greater distinction from variables [685](https://github.com/NOAA-EMC/UPP/blob/0044f64dc75a9c711be79d1205c55650cf1b7291/doc/UserGuide/tables/UPP_GRIB2_Table_byID.csv#L650) and [744](https://github.com/NOAA-EMC/UPP/blob/0044f64dc75a9c711be79d1205c55650cf1b7291/doc/UserGuide/tables/UPP_GRIB2_Table_byID.csv#L699).